config/kubernetes/sig-docs/teams.yaml

```diff
@@ -300,6 +300,20 @@ teams:
     - MaxymVlasov
     - Potapy4
     privacy: closed
+  sig-docs-bn-owners:
+    description: Admins for Bengali content
+    members:
+    - mitul3737
+    - Mouly22
+    - ikramulkayes
+    privacy: closed
+  sig-docs-es-reviews:
```

Comment:
possible nit?
```suggestion
  sig-docs-bn-reviews:
```

Reply:
Yes, need to change.

Reply:
![p1](https://user-images.githubusercontent.com/57193846/152724630-a90f0bd8-c686-4b69-8943-ac94b20a9f04.png)

Can you have a look now @palnabarun ?

Reply:
Thanks! :)